perf/perf_metric.py

```diff
@@ -82,12 +82,11 @@ def setUp(self):
 
     def _run_cmd(self, option):
         for line in self.list_of_metric_events:
-            cmd = "perf stat %s %s sleep 1" % (option, line)
-            rc, op = process.getstatusoutput(cmd, ignore_status=True,
-                                             shell=True, verbose=True)
+            cmd = "perf stat %s %s -C 0 sleep 1" % (option, line)
+            op = process.run(cmd, ignore_status=True, shell=True, verbose=True)
+            output = op.stdout.decode() + op.stderr.decode()
```

Comment:
make use of % operator to concatenate and also avoid  redundant decode

output = ('%s %s' % (op.stdout, op.stderr)).decode()

Reply:
The %s formatting operator performs string interpolation, converting the byte strings to string representations and concatenating them, so we cant use decode() with it.
```
>>> output = ('%s %s' % (op.stdout, op.stderr)).decode()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'str' object has no attribute 'decode'
```
We can use either of this two
```
output = '%s %s' % (op.stdout.decode(), op.stderr.decode())
output = (op.stdout + op.stderr).decode()
```

Reply:
Alright, Please go with output = (op.stdout + op.stderr).decode()

Reply:
I have done the changes and pushed.